applications/serial_lte_modem/src/slm_uart_handler.c

```diff
@@ -144,15 +137,13 @@ int slm_uart_rx_disable(void)
 		k_sleep(K_MSEC(10));
 	}
 
-	err = uart_rx_disable(uart_dev);
+	err = uart_rx_disable(slm_uart_dev);
 	if (err) {
 		LOG_ERR("UART RX disable failed: %d", err);
+		atomic_set(&recovery_state, RECOVERY_IDLE);
```

Comment:
Unsure if the uart is actually in operationable state in here, but I guess that setting this does not harm.

Reply:
Yeah, the hypothetical reasoning behind this is that if disabling fails, then it's still running, so set `recovery_state` back to a running state.